src/components/ProductContainer/ProductContainer.tsx

```diff
@@ -66,11 +69,8 @@ const ProductContainer = ({allProducts, filteredProducts, savedProducts, searchi
       })
     } 
     
-  let featuredProductCards;
-  if(featuredProducts.length) {
-  featuredProductCards = getProductCards(featuredProducts)
-  }
   
+  const featuredProductCards = getProductCards(featuredProducts)
```

Comment:
I see we no longer have that conditional checking for featuredProductCards.length before creating this variable. Why is that? I ask because  I still (very rarely) see that sometimes the 'Featured Items' section will be empty. I wonder that if this conditional was still here that the rarity of it being empty wouldn't happen anymore.

Reply:
I took it out because it was one of the things I originally thought would fix the cypress testing, but it was inconsistent so after I realized that our fix for cypress testing was completely different (the conditional that skips the recursion altogether) I took it out since it was really just unnecessary code that wasn't doing anything. Going to keep trying to figure out why sometimes featured items are empty.